src/tripCard/TripCard.tsx

```diff
@@ -190,7 +211,11 @@ export const TripCard = (props: TripCardProps) => {
                   <div className="kirk-tripCard-bottom-left">
                     {driver && (
                       <div className="kirk-tripCard-driver">
-                        <Avatar className="kirk-tripCard-avatar" image={driver.avatarUrl} />
+                        <Avatar
+                          className="kirk-tripCard-avatar"
+                          image={driver.avatarUrl}
+                          alt={driver.firstName}
```

Comment:
I wouldn't add the alt here, since it's repeating the driver first name displayed juste after, and does not provide additional information.

Reply:
From MDN:

> Omitting alt altogether indicates that the image is a key part of the content and no textual equivalent is available. Setting this attribute to an empty string (alt="") indicates that this image is not a key part of the content (it’s decoration or a tracking pixel), and that non-visual browsers may omit it from rendering. Visual browsers will also hide the broken image icon if the alt is empty and the image failed to display.
> https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img#attr-alt

Maybe we should use `alt=""` as the avatar don't add relevant information, the user's name is enough.
What do you think?

cc @nickoola

Reply:
It looks like it's already the default value in `Avatar`.

Reply:
Yes I forgot to mention it but `Avatar` already handles it :)